packages/api/src/mapping/PropertyValueMapping.ts

```diff
@@ -42,6 +42,7 @@ export interface PropertyValueWireToClient {
 
   numericTimeseries: TimeSeriesProperty<number>;
   stringTimeseries: TimeSeriesProperty<string>;
+  sensorTimeseries: TimeSeriesProperty<string>;
```

Comment:
Sensor timeseries can come over the wire as doubles or strings, so just serializing as string